content/development/how-to-write-a-loader.md

```diff
@@ -17,6 +19,28 @@ The loader is expected to give back one or two values. The first value is a resu
 
 In the complex case, when multiple loaders are chained, only the last loader gets the resource file and only the first loader is expected to give back one or two values (JavaScript and SourceMap). Values that any other loader give back are passed to the previous loader.
 
+In other words the loading order for chained loaders goes from right to left or bottom to top. 
+For example: lets say you have two loaders that go by the name of `fooLoader` and `barLoader`. You would like to execute `fooLoader` and then pass the result of the transformation from `fooLoader` finally to `barLoader`. 
+
+You would add the following in your config file (assuming that both loaders are already defined):
+
+``` javascript
+module: {
+         rules: [{
+                  test: /\.js/,
+                  use: [{ loader: 'barLoader' }, { loader: 'fooLoader' }]
+                ]
+         }
+```
+
+Note that webpack currently only searches in your node modules folder for loaders. If these loaders are defined outside your node modules folder you would need to use the `resolveLoader` property to get webpack to include your loaders. For example lets say you have your custom loaders included in a folder called `loaders`. You would have to add the following to your config file: 
+
+``` javascript
+resolveLoader: {
+modules: ['node_modules', path_resolve(__dirname, 'loaders')
```

Comment:
Should be indented two spaces (one tab if your editor is listening to the `.editorconfig`).